examples/basic/android/app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +20,7 @@
       android:icon="@mipmap/ic_launcher"
       android:theme="@style/AppTheme"
       tools:node="replace"
+      android:usesCleartextTraffic="true"
```

Comment:
why this?

Reply:
I think this is a sec on Android 9.

“Fix Cleartext Traffic Error in Android 9 Pie” by Rommer Cañete https://link.medium.com/QtWKFr0Q0W